src/GraphRequest.ts

```diff
@@ -367,7 +372,15 @@ export class GraphRequest {
 		let rawResponse: Response;
 		const middlewareControl = new MiddlewareControl(this._middlewareOptions);
 		this.updateRequestOptions(options);
-		const customHosts = this.config?.customHosts;
+        const customHosts = this.config?.customHosts;
+		const requestInfo = new RequestInformation();
```

Comment:
assuming we're manipulating a BaseBearerTokenAuthenticationProvider and this provides a property for the access token provider, you shouldn't have to instantiate a request information at all for arbitrary requests